beater/burrowbeat.go

```diff
@@ -11,15 +11,15 @@ import (
 	"github.com/elastic/beats/libbeat/beat"
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
-	"github.com/elastic/beats/libbeat/publisher"
+	pub "github.com/elastic/beats/libbeat/publisher"
```

Comment:
why change this line and line 22? This gets us absolutely nothing from what I see.